tests/unit/Collectors/JobCollectorTest.php

```diff
@@ -56,14 +58,18 @@ protected function _before()
         $this->agentMock = Mockery::mock(Agent::class);
         $this->configMock = Mockery::mock(Config::class);
 
+        $this->configMock = Mockery::mock(Config::class);
+
         $requestStartTimeMock = Mockery::mock(RequestStartTime::class);
         $requestStartTimeMock->shouldReceive('setStartTime');
         $requestStartTimeMock->shouldReceive('microseconds')->andReturn(1000.0);
 
         $this->collector = new JobCollector(
```

Comment:
The merge from master didn't go really well, by bad.

```suggestion
        $eventCounter = new EventCounter();
        $eventClock = new EventClock();

        $this->collector = new JobCollector(
```

Reply:
Live coding on GitHub, this is a new dimension.